mcnp/mcnp_funcs.cpp

```diff
@@ -476,16 +476,38 @@ void write_lcad_old(std::ofstream &lcadfile)
   int num_surfs = DAG->num_entities( 2 );
 
   int cmat = 0;
-  double crho, cimp = 1.0;
+  double crho = 0;
+  double cimp_n = 0, cimp_p = 0, cimp_e = 0;
+
+  // Detect which importances are used so all cells, including implicit
+  // complement and graveyard, have these importances
+  bool imp_n_needed = false, imp_p_needed = false, imp_e_needed = false;
+  for( int i = 1; i <= num_cells; ++i ) {
+    moab::EntityHandle vol = DAG->entity_by_index( 3, i );
+    if( DAG->has_prop( vol, "imp.n" )) {
```

Comment:
Is there a way to search for the existence of this tag directly in MOAB without looping through the volumes?  (Maybe not)